tools/expotools/src/prebuilds/Prebuilder.ts

```diff
@@ -0,0 +1,240 @@
+        GCC_GENERATE_DEBUGGING_SYMBOLS: false,
+        ...settings,
+      })
+    );
+  }
+
+  // Print binary sizes
+  const binarySizes = frameworks.map((framework) =>
+    chalk.magenta((framework.binarySize / 1024 / 1024).toFixed(2) + 'MB')
+  );
+  logger.log('   Binary sizes:', binarySizes.join(', '));
+
+  logger.log('   Merging frameworks to', chalk.magenta(`${xcodeProject.name}.xcframework`));
+
+  // Merge frameworks into universal xcframework
+  await xcodeProject.buildXcframeworkAsync(frameworks, settings);
+}
+
+/**
+ * Removes all temporary files that we generated in order to create `.xcframework` file.
+ */
+export async function cleanTemporaryFilesAsync(xcodeProject: XcodeProject) {
+  logger.log('   Cleaning up temporary files');
+
+  const pathsToRemove = [`${xcodeProject.name}.xcodeproj`, INFO_PLIST_FILENAME];
+
+  await Promise.all(
+    pathsToRemove.map((pathToRemove) => fs.remove(path.join(xcodeProject.rootDir, pathToRemove)))
+  );
+}
+
+/**
+ * Generates Xcode project based on the podspec of given package.
+ */
+export async function generateXcodeProjectSpecAsync(pkg: Package): Promise<XcodeProject> {
+  const podspec = await pkg.getPodspecAsync();
+
+  if (!podspec) {
+    throw new Error('Given package is not an iOS project.');
+  }
+
+  logger.log('   Generating Xcode project spec');
+
+  const spec = await createSpecFromPodspecAsync(podspec, async (dependencyName) => {
+    const frameworkPath = await findFrameworkForProjectAsync(dependencyName);
+
+    if (frameworkPath) {
+      return {
+        framework: frameworkPath,
+        link: false,
+        embed: false,
+      };
+    }
+    return null;
+  });
+
+  const xcodeprojPath = await generateXcodeProjectAsync(
+    path.join(pkg.path, pkg.iosSubdirectory),
+    spec
+  );
+  return await XcodeProject.fromXcodeprojPathAsync(xcodeprojPath);
+}
+
+/**
+ * Removes prebuilt `.xcframework` files for given packages.
+ */
+export async function cleanFrameworksAsync(packages: Package[]) {
+  for (const pkg of packages) {
+    const xcFrameworkFilename = `${pkg.podspecName}.xcframework`;
+    const xcFrameworkPath = path.join(pkg.path, pkg.iosSubdirectory, xcFrameworkFilename);
+
+    if (await fs.pathExists(xcFrameworkPath)) {
+      await fs.remove(xcFrameworkPath);
+    }
+  }
+}
+
+/**
+ * Checks whether given project name has a framework (GoogleSignIn, FBAudience) and returns its path.
+ */
+async function findFrameworkForProjectAsync(projectName: string): Promise<string | null> {
+  const searchNames = new Set([
+    projectName,
+    projectName.replace(/\/+/, ''), // Firebase/MLVision -> FirebaseMLVision
+  ]);
+
+  for (const name of searchNames) {
+    const cwd = path.join(PODS_DIR, name);
+
+    if (await fs.pathExists(cwd)) {
+      const paths = await glob(`**/*.framework`, {
+        cwd,
+      });
+
+      if (paths.length > 0) {
+        return path.join(cwd, paths[0]);
+      }
+    }
+  }
+  return null;
+}
+
+//#endregion
```

Comment:
🤔

Reply:
Ah, sorry, I've been looking for something like `#pragma mark` or `//region` in Android Studio but for VSCode. Ultimately I wanted to bring this up on the meeting.